packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -175,11 +179,44 @@ export interface ChatMessageProps
 
   /** Positions an actionMenu slot in "fixed" mode. */
   unstable_overflow?: boolean;
+
+  /**
+   * Properties that are only available to the v2 (refresh) version of
+   * ChatMessage because they only have an effect there. Enabling this property
+   * will opt into the new layout.
+   */
+  v2?: ChatMessageV2Props;
+}
+
+interface ChatMessageV2Props {
+  /** Indicates whether the message is in a failure state. */
+  failed?: boolean;
+
+  /** Label that describes message importance. */
+  importantLabel?: React.ReactNode;
+
+  /** Additional label slot for message importance, scheduled messages, etc. */
+  infoLabel?: React.ReactNode;
+
+  /**
+   * Hook that can be used to customize the element rendered for the message bubble.
+   * This is required for external, platform-specific implementations that need to
+   * wrap the bubble in order to inject animations et al.
+   */
+  customizeBubbleElement?(element: React.ReactElement): React.ReactElement;
+
+  /** Slot for elements that sit next to the message bubble. */
+  bubbleInset?: React.ReactNode;
```

Comment:
Should we use Shorthand as type instead of raw ReactNode?

Reply:
Should we? What's the benefit of using a shorthand here rather than just accepting a renderable node?

Reply:
@ling1726 any opinions on keeping with the convention for these older components?